### PR TITLE
Session management & Logout 

### PR DESCRIPTION
**Pull request by miguelhenaoa**
# Summary
- Created component header
- Created route book and component book
- Add access token to LocalStorage
- Remove access token from LocalStorage

# Trello Card
https://trello.com/c/Vc3sJWQo

# Screenshot
image
image

### CODE REVIEW

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="header">
```
Comment (ebedoyaalzate) [inline src/app/components/header/header.component.html]: Use header tag

File: src/app/screens/books/books.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  templateUrl: './books.component.html',
+  styleUrls: ['./books.component.scss']
+})
+export class BooksComponent {
+
+  constructor(private router: Router) { }
+
+  logout() {
+    localStorage.removeItem('ACCESS_TOKEN');
+    this.router.navigate(['auth/login']);
```
Comment (ebedoyaalzate) [inline src/app/screens/books/books.component.ts]: Is necessary logout from books component?

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: src/app/components/header/header.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should session logout and remove access token', () => {
+    const button: HTMLElement = fixture.debugElement.nativeElement.querySelector('#btn-logout');
+    button.click();
+
+    expect(localStorage.getItem('ACCESS_TOKEN')).toBeNull();
```
Comment (ebedoyaalzate) [inline src/app/components/header/header.component.spec.ts:41]: Add items to localstorage before delete it, for verify the correct use.

File: src/app/screens/auth/sign-in/sign-in.component.ts [hunk 4]
```diff
@@ -47,7 +48,8 @@ export class SignInComponent implements OnInit {
   signIn() {
     this.invalidCredetials = false;
     this.service.signIn(this.form.value as SignIn).subscribe((res) => {
-     console.log(res.headers.get('access-token'));
+     localStorage.setItem('ACCESS_TOKEN', res.headers.get('access-token'));
+     this.router.navigate(['books']);
     }, error => {
       console.error(error);
```
Comment (ebedoyaalzate) [inline src/app/screens/auth/sign-in/sign-in.component.ts]: delete this console.log

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: src/app/components/header/header.component.html [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<header class="header">
+  <img class="logo-wolox" src="assets/images/logo_wolox.svg" alt="Logo Wolox">
+  <a id="btn-logout" class="btn btn-logout" (click)="sessionLogout()">Logout</a>
```
Comment (CrisVega08) [inline src/app/components/header/header.component.html]: Use btn

File: src/app/components/header/header.component.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
+@import "src/styles/lib/colors";
+
+.header {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 4px 10px 0 rgba(0,0,0,0.5);
```
Comment (CrisVega08) [inline src/app/components/header/header.component.scss]: use variable for color

File: src/app/components/header/header.component.scss [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+.logo-wolox {
+  height: 70px;
+  margin-left: 18%;
+  width: 210px;
+}
+
+.btn-logout {
+  height: 18px;
+  color: #000;
```
Comment (CrisVega08) [inline src/app/components/header/header.component.scss]: Use variable

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Review by ebedoyaalzate — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**